rfc047-release-process.md

```diff
@@ -44,40 +44,51 @@ Additional labels for pre-release and build metadata are available as extensions
 Chef Client specific clarifications:
 
 ```
-Given a version number MAJOR.MINOR.PATCH, increment the:
+Given a version number MAJOR.MINOR.BUILD:
 
-MAJOR version when the release will break existing functionality
-MINOR version when you add any new features
-PATCH version when you are only fixing bugs, typically regressions
+Increment the MAJOR version when the release will break existing functionality
+Increment the MINOR version when you add any new features
+The BUILD version is increased automatically by the build system
```

Comment:
I know you touch on it briefly below but do we want to mention that:

* `MINOR` and `BUILD` are reset to zero when `MAJOR` is incremented.
* `BUILD` is reset to zero when `MINOR` is incremented.